ios/FluentUI/Notification/MSFNotificationTokens.swift

```diff
@@ -15,6 +15,25 @@ import UIKit
     case dangerToast
     case warningToast
 
+    public var styleString: String {
```

Comment:
This should be added in an extension of the MSFNotificationStyle enum in the demo app (example in the link below):

https://github.com/microsoft/fluentui-apple/blob/7fbc76a9a60e69d84fd9026b134fc1c15725a27f/ios/FluentUI.Demo/FluentUI.Demo/Demos/ActivityIndicatorDemoController.swift#L217